### ISSUE TITLE
Improve performance when using plugins and solve small bug

### ISSUE THREAD

**Issue by rapphil** [shirosaidev/diskover#16]
For every file that is analysed using get_file_meta, all the plugins are being reloaded. We should load the plugins only once.

**Comment by rapphil** [comment]
I came across a performance issue when I was running diskover with plugins. After profiling the code with cProfile, noticed that greate part of the time was spent in the load_plugin function.

It happens that, if the module was already imported, it will be reloaded. https://docs.python.org/2/library/imp.html#imp.load_module

Maybe this a side effect of using PYTHONDONTWRITEBYTECODE, which I use. In my case the performance was significantly improved when loading the plugins only once at the beginning.

**Comment by shirosaidev** [comment]
This was merged into the last release of diskover (v1.4.2), thanks for the code improvement.
